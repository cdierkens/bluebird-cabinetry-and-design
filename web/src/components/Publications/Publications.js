import React from "react";
import styles from "./Publications.module.css";

const Publications = () => {
  return (
    <div className={styles.PublicationsContainer}>
      <h2 className={styles.PublicationsH2}>Publications</h2>
    </div>
  );
};

export default Publications;
