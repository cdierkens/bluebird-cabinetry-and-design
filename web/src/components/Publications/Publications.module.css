.PublicationsContainer {
  @apply py-8 grid grid-cols-12 bg-blue-dark;
}

.PublicationsH2 {
  @apply text-center text-white col-span-12;
}
