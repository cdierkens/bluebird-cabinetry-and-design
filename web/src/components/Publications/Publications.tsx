import React from "react";
import { BestIcon, ShilowCabinetryIcon } from "../../icons";
import Container from "../Container";

const Publications: React.FC = () => {
  return (
    <div className="bg-blue-dark py-6">
      <Container>
        <h2 className="text-center text-white m-0 mb-3">Publications</h2>

        <div className="flex justify-around flex-wrap lg:w-2/3 m-auto">
          <div className="col-span-2 sm:col-span-3 md:col-start-2 text-center m-3">
            <a
              className="text-white hover:text-gold inline-block"
              href="/shiloh-brochure-2017.pdf"
            >
              <ShilowCabinetryIcon className="h-16 mb-1" />

              <span className="font-sans text-lg font-medium block">
                2017 National
                <br />
                Shiloh Brochure
              </span>
            </a>
          </div>

          <div className="col-span-2 sm:col-span-3 text-center m-3">
            <a
              className="text-white hover:text-gold inline-block"
              href="/best-fine-homes-magazine-fall-2017.pdf"
            >
              <BestIcon className="h-16 mb-1" />

              <span className="font-sans text-lg font-medium block">
                BEST Fine Homes
                <br />
                Magazine
              </span>
            </a>
          </div>

          <div className="col-span-2 sm:col-span-3 text-center m-3">
            <a
              className="text-white hover:text-gold inline-block"
              href="/shiloh-brochure-2019.pdf"
            >
              <ShilowCabinetryIcon className="h-16 mb-1" />

              <span className="font-sans text-lg font-medium block">
                2019 National
                <br />
                Shiloh Brochure
              </span>
            </a>
          </div>
        </div>
      </Container>
    </div>
  );
};

export default Publications;
