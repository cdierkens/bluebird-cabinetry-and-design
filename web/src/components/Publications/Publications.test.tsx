import { render } from "@testing-library/react";
import React from "react";
import Publications from "./Publications";

it("matches snapshot", () => {
  const { asFragment } = render(<Publications />);

  expect(asFragment()).toMatchSnapshot();
});
