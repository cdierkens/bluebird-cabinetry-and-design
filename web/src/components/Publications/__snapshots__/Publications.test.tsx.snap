// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="bg-blue-dark py-6"
  >
    <div
      class="Container"
    >
      <h2
        class="text-center text-white m-0 mb-3"
      >
        Publications
      </h2>
      <div
        class="flex justify-around flex-wrap lg:w-2/3 m-auto"
      >
        <div
          class="col-span-2 sm:col-span-3 md:col-start-2 text-center m-3"
        >
          <a
            class="text-white hover:text-gold inline-block"
            href="/shiloh-brochure-2017.pdf"
          >
            <test-file-stub
              classname="p-1 fill-current stroke-current inline-block h-16 mb-1"
            />
            <span
              class="font-sans text-lg font-medium block"
            >
              2017 National
              <br />
              Shiloh Brochure
            </span>
          </a>
        </div>
        <div
          class="col-span-2 sm:col-span-3 text-center m-3"
        >
          <a
            class="text-white hover:text-gold inline-block"
            href="/best-fine-homes-magazine-fall-2017.pdf"
          >
            <test-file-stub
              classname="p-1 fill-current stroke-current inline-block h-16 mb-1"
            />
            <span
              class="font-sans text-lg font-medium block"
            >
              BEST Fine Homes
              <br />
              Magazine
            </span>
          </a>
        </div>
        <div
          class="col-span-2 sm:col-span-3 text-center m-3"
        >
          <a
            class="text-white hover:text-gold inline-block"
            href="/shiloh-brochure-2019.pdf"
          >
            <test-file-stub
              classname="p-1 fill-current stroke-current inline-block h-16 mb-1"
            />
            <span
              class="font-sans text-lg font-medium block"
            >
              2019 National
              <br />
              Shiloh Brochure
            </span>
          </a>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
